pkg/ocm/clusters.go

```diff
@@ -114,6 +113,10 @@ type Spec struct {
 	// Disable SCP checks in the installer by setting credentials mode as mint
 	DisableSCPChecks *bool
 
+	// Non-STS
+	AWSAccessKey *aws.AccessKey
+	AWSCreator   *aws.Creator
```

Comment:
@ciaranRoche one question - the other `pkg/ocm` client calls take this in as a parameter - I know you mentioned putting in the `struct` here, but wanted to check that there wasn't a preference one way or the other